stores/useWalletStore.tsx

```diff
@@ -192,7 +198,7 @@ const INITIAL_PROPOSAL_STATE = {
 
 const useWalletStore = create<WalletStore>((set, get) => ({
   connected: false,
-  connection: INITIAL_CONNECTION_STATE,
+  connection: getConnectionConfig('mainnet'),
```

Comment:
making this as default means the UI will start to render with mainnet.

If something else is set via query param, it will immediately re-render. A fix for this is to make connection optionally undefined. This can be done in a follow-up PR since its a messier change and we may want to consider another approach like making connection its own context

Reply:
The extra re-render for non mainnet is ok for now. Good triage!

Reply:
It's ok to re-render for non mainnet. Good Triage! :+1: